### DISCUSSION TITLE
Enable x299 + 7800x resizable bar

### DISCUSSION THREAD

**Discussion by timido22** [xCuri0/ReBarUEFI#178] [Q&A] [answered]
I have an x299 Asrock steel legend and am trying to understand. How need patch bios? My proc. 7800x + rtx3090 palit
![image](https://github.com/xCuri0/ReBarUEFI/assets/154282702/ac5d5da2-27ac-43b1-9b06-0f0d940b8171)
which generation needs to choose?

**Comment by xCuri0** [comment, marked as answer]
none of them, you dont need to do usb 3.0 patch

**Reply by timido22** [reply]
ok thanks, but I see the next
![image](https://github.com/xCuri0/ReBarUEFI/assets/154282702/2a4c5bef-297e-43b2-9295-22a4e80836fa)

and in this text, I see that. What need to do after?

![image](https://github.com/xCuri0/ReBarUEFI/assets/154282702/846b3221-1330-4852-80f2-233ae3da9b86)
